ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto

```diff
@@ -8,6 +8,7 @@ package com.daml.ledger.api.v1;
 import "com/daml/ledger/api/v1/commands.proto";
 import "com/daml/ledger/api/v1/transaction.proto";
 import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
```

Comment:
@nicu-da : could this be a spurious import? The change does not mention any wrappers in its diff.

Reply:
You are right. I removed the usage of the wrapper but kept the import. Removing it.